playbooks/appsemblerPlaybooks/eucalyptus_enterprise_deploy.yml

```diff
@@ -112,3 +112,21 @@
     - role: nginx
       nginx_sites:
       - certs
+
+- name: Install Scampersand Reporting
+  hosts: "edxapp-server-0"
+  sudo: True
+  gather_facts: True
+  vars:
+    POSTGRESQL_AGGREGATION_INSTALL_CRON: true
+  roles:
+    - postgresql_aggregation
+
+- name: Install Scampersand Reporting
+  hosts: "edxapp-server-1"
```

Comment:
Sure, it's fine for this particular thing :) In general, it becomes harder to manage servers the more differences we have.